### DISCUSSION TITLE
Add embedded help/links to external documentation in all contexts

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5773] [Feature Requests]
As seen in the App Resources page, there needs to be some context-sensitive documentation links that can be triggered by a key press or by some information icon on forms, in queries, tools, etc.

_Similar to the F1 help in Specify 6._

**Requested By:** Barcelona and others

**Comment by grantfitzsimmons** [comment]
<img width="274" alt="Screenshot 2023-05-25 at 12 51 40 PM" src="https://github.com/specify/specify7/assets/37256050/40cd674b-8901-429f-9e2a-11bd13be5a68">

Google does a fairly good job at this

**Comment by maxpatiiuk** [comment]
Embedded discourse articles would be helpful
